.github/workflows/haskell.yml

```diff
@@ -79,6 +79,15 @@ jobs:
             cabal_version: 3.8.1.0
             disable_sdist_build: "y"
             ignore_error: false
+          - name: 9.4.2-fusion-inspection
+            ghc_version: 9.4.2
+            runner: ubuntu-latest
+            build: cabal
+            cabal_version: 3.8.1.0
+            disable_sdist_build: "y"
+            cabal_project: cabal.project
+            cabal_build_options: "--flag fusion-plugin --flag inspection"
+            ignore_error: false
```

Comment:
Let's remove the other 9.4.2 test, there are 12 CIs right now, should try to reduce.